### PR TITLE
My new branch

### PR DESCRIPTION
**Pull request by katiamoklokova**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,5 @@
 this is a supposed to be description
 
 ![pizza](images_folder/pep_pizza_BIG.jpg)
+
```
Comment (pachkovska) [inline README.md]: looks good

File: README.md [hunk 2]
```diff
@@ -3,3 +3,5 @@
...
+I'm going to edit this now!
```
Comment (pachkovska) [inline README.md]: requesting some changes

File: colors.md [hunk 3]
```diff
@@ -1 +1 @@
-[new image] (images_folder/incolour-packshot.jpg)
+[new image](images_folder/incolour-packshot.jpg)
```
Comment (pachkovska) [inline colors.md]: Can you add some description to your image? Thanks

Comment (pachkovska) [inline colors.md]: give me more images.

**Comment by katiamoklokova** [comment]
I think I added the changes?

**Comment by katiamoklokova** [comment]
Hopefully you can see them

**Comment by pachkovska** [comment]
merged

**Pull request merged**